builder/lxc/step_lxc_create.go

```diff
@@ -88,6 +101,16 @@ func (s *stepLxcCreate) Run(state multistep.StateBag) multistep.StepAction {
 		errorHandler(err)
 		return multistep.ActionHalt
 	}
+
+	for _, sidedisk := range config.SidediskFolders {
+		ui.Say(fmt.Sprintf("Loading sidedisk \"%s\" to %s", sidedisk.Archive, sidedisk.Dest))
+		err = s.loadSidedisk(config.ContainerName, sidedisk.Archive, sidedisk.Dest)
```

Comment:
We have a `rootfs` path from starting the container that we can use in the `loadSidedisk` function instead of constructing the rootfs from the container name.